daemon/api.go

```diff
@@ -157,6 +164,46 @@ func sysInfo(c *Command, r *http.Request) Response {
 	return SyncResponse(m)
 }
 
+func loginUser(c *Command, r *http.Request) Response {
+	var loginData struct {
+		Username string `json:"username"`
+		Password string `json:"password"`
+		Otp      string `json:"otp"`
+	}
+
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&loginData); err != nil {
+		return BadRequest("can't decode login data from request body: %v", err)
```

Comment:
s/can't/cannot/

failed to, can't, can not, couldn't, could not, unable to => _cannot_